litmus-portal/authentication/pkg/utils/configs.go

```diff
@@ -2,14 +2,24 @@ package utils
 
 import (
 	"os"
+	"strconv"
 )
 
 var (
-	JwtSecret     = os.Getenv("JWT_SECRET")
-	AdminName     = os.Getenv("ADMIN_USERNAME")
-	AdminPassword = os.Getenv("ADMIN_PASSWORD")
-	DBUrl         = os.Getenv("DB_SERVER")
-	DBUser        = os.Getenv("DB_USER")
-	DBPassword    = os.Getenv("DB_PASSWORD")
-	DBName        = "auth"
+	JwtSecret         = os.Getenv("JWT_SECRET")
+	AdminName         = os.Getenv("ADMIN_USERNAME")
+	AdminPassword     = os.Getenv("ADMIN_PASSWORD")
+	DBUrl             = os.Getenv("DB_SERVER")
+	DBUser            = os.Getenv("DB_USER")
+	DBPassword        = os.Getenv("DB_PASSWORD")
+	JWTExpiryDuration = getEnvAsInt("JWT_EXPIRY_MINS", 1440)
+	DBName            = "auth"
```

Comment:
Why is this DBName in config and the collection and field names in the main file? Also the port can be moved to config @DarthBenro008

Reply:
Oops :sweat_smile: , I had taken the reference from the current authentication server, hence followed the same. It has been moved to configs @ishangupta-ds